content/src/fieldConfig/tax-clearance-certificate-download.json

```diff
@@ -0,0 +1,7 @@
+{
+  "taxClearanceCertificateDownload": {
+    "headerTwoLabel": "Your Certificate is Ready!",
+    "calloutText": "Download your certificate now, or you’ll have to request it again.",
+    "buttonText": "Tax Clearance Certificate (PDF)"
+  }
+}
```

Comment:
Is CMS search still broken? We really need to fix that because adding additional items to the CMS is only going to make fixing this in the future more difficult for the person who addresses this.